<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>TransferForm</class>
  <widget class="QDialog" name="TransferForm">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>532</width>
        <height>251</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Twinkle - Transfer</string>
    </property>
    <layout class="QVBoxLayout">
      <item>
        <widget class="Q3GroupBox" name="transferGroupBox">
          <property name="title">
            <string>Transfer call to</string>
          </property>
          <layout class="QHBoxLayout">
            <item>
              <widget class="QLabel" name="toLabel">
                <property name="text">
                  <string>&amp;To:</string>
                </property>
                <property name="buddy" stdset="0">
                  <cstring>toLineEdit</cstring>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLineEdit" name="toLineEdit">
                <property name="whatsThis" stdset="0">
                  <string>The address of the person you want to transfer the call to. This can be a full SIP address like &lt;b&gt;sip:example@example.com&lt;/b&gt; or just the user part or telephone number of the full address. When you do not specify a full address, then Twinkle will complete the address by using the domain value of your user profile.</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QToolButton" name="addressToolButton">
                <property name="focusPolicy">
                  <enum>Qt::TabFocus</enum>
                </property>
                <property name="text">
                  <string/>
                </property>
                <property name="shortcut">
                  <string>F10</string>
                </property>
                <property name="icon">
                  <iconset>:/icons/images/kontact_contacts.png</iconset>
                </property>
                <property name="toolTip" stdset="0">
                  <string>Address book</string>
                </property>
                <property name="whatsThis" stdset="0">
                  <string>Select an address from the address book.</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="Q3ButtonGroup" name="typeButtonGroup2">
          <property name="title">
            <string>Type of transfer</string>
          </property>
          <layout class="QVBoxLayout">
            <item>
              <widget class="QRadioButton" name="basicRadioButton">
                <property name="text">
                  <string>&amp;Blind transfer</string>
                </property>
                <property name="shortcut">
                  <string>Alt+B</string>
                </property>
                <property name="checked">
                  <bool>true</bool>
                </property>
                <property name="whatsThis" stdset="0">
                  <string>Transfer the call to a third party without contacting that third party yourself.</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QRadioButton" name="consultRadioButton">
                <property name="text">
                  <string>T&amp;ransfer with consultation</string>
                </property>
                <property name="shortcut">
                  <string>Alt+R</string>
                </property>
                <property name="whatsThis" stdset="0">
                  <string>Before transferring the call to a third party, first consult the party yourself.</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QRadioButton" name="otherLineRadioButton">
                <property name="text">
                  <string>Transfer to other &amp;line</string>
                </property>
                <property name="shortcut">
                  <string>Alt+L</string>
                </property>
                <property name="whatsThis" stdset="0">
                  <string>Connect the remote party on the active line with the remote party on the other line.</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <spacer name="spacer24">
          <property name="sizeHint">
            <size>
              <width>20</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <spacer name="spacer23">
              <property name="sizeHint">
                <size>
                  <width>121</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="okPushButton">
              <property name="text">
                <string>&amp;OK</string>
              </property>
              <property name="shortcut">
                <string>Alt+O</string>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="cancelPushButton">
              <property name="text">
                <string>&amp;Cancel</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <includes>
    <include location="local">qstring.h</include>
    <include location="local">sockets/url.h</include>
    <include location="local">ui_getaddressform.h</include>
    <include location="local">user.h</include>
    <include location="local">protocol.h</include>
    <include location="local">phone.h</include>
  </includes>
  <connections>
    <connection>
      <sender>okPushButton</sender>
      <signal>clicked()</signal>
      <receiver>TransferForm</receiver>
      <slot>validate()</slot>
    </connection>
    <connection>
      <sender>cancelPushButton</sender>
      <signal>clicked()</signal>
      <receiver>TransferForm</receiver>
      <slot>reject()</slot>
    </connection>
    <connection>
      <sender>addressToolButton</sender>
      <signal>clicked()</signal>
      <receiver>TransferForm</receiver>
      <slot>showAddressBook()</slot>
    </connection>
    <connection>
      <sender>otherLineRadioButton</sender>
      <signal>toggled(bool)</signal>
      <receiver>TransferForm</receiver>
      <slot>setOtherLineAddress(bool)</slot>
    </connection>
  </connections>
</ui>
